src/fiat-connect-client.ts

```diff
@@ -43,15 +44,22 @@ export class FiatConnectClientImpl implements FiatConnectApiClient {
   signingFunction: (message: string) => Promise<string>
   _sessionExpiry?: Date
   fetchImpl: typeof fetch
+  cookieJar: CookieJar
 
   constructor(
     config: FiatConnectClientConfig,
     signingFunction: (message: string) => Promise<string>,
     fetchImpl: typeof fetch,
+    cookieJar?: CookieJar.Serialized,
```

Comment:
this needs to be included in FiatConnectApiClient. Wonder if we should also make signingFunction optional. cc @jophish